app/helpers/column_selector_helper.rb

```diff
@@ -0,0 +1,15 @@
+module ColumnSelectorHelper
+  def get_selected_columns
+    User.current.table_preferences.find_by_name(auto_complete_controller_name).columns
+  end
+
+  def mount_column_selector
+    url = send("api_users_path") + "/#{User.current[:id]}/table_preferences/#{auto_complete_controller_name}"
+    react_component("ColumnSelector", data: {
+                      url: url,
+                      controller: auto_complete_controller_name,
+                      selected: get_selected_columns,
+                      columns: ["Power", "Name", "Operating system", "Model", "Owner", "Host group", "Last report", "Comment"],
```

Comment:
Should those be translated?

Reply:
Also Oleh's PR expects the column names to be lowercase

Reply:
This is just temporary solution, I think instead of this will be used https://github.com/theforeman/foreman/blob/ff625deddc1de77d0572fdcafa2255f19c47b17f/app/registries/foreman/selectable_columns/storage.rb#L26